### PR TITLE
Add jira flags

### PR DESCRIPTION
**Pull request by wernerb**
Relies on https://github.com/terraform-providers/terraform-provider-gitlab/pull/273 

The jira service has some extra flags that reduce the spamminess of the service.

### CODE REVIEW

**Comment by ringods** [comment]
Hello @wernerb, PR #273 is merged. Can you rebase this work on `master` first?

**Comment by wernerb** [comment]
@ringods rebased. thank you!

**Comment by ringods** [comment]
@wernerb would you still be able to add 1 or more acceptance tests?

**Comment by oboukili** [comment]
Great work, just chiming about an issue: it seems there is a drift with the `comment_on_event_enabled` and `commit_events`attributes.

Given the following resource:

```hcl
  resource "gitlab_service_jira" "oms" {
      comment_on_event_enabled = true
      commit_events            = true
      jira_issue_transition_id = "31"
      merge_requests_events    = true
      password                 = "foo"
      project                  = "1234567"
      project_key              = "PROJ"
      url                      = "https://jira.url"
      username                 = "test@test.com"
  }
```
Every run tries to modify the state:

```hcl
  ~ resource "gitlab_service_jira" "oms" {
        active                   = true
      ~ comment_on_event_enabled = false -> true
      ~ commit_events            = false -> true
        created_at               = "2020-04-02 10:32:47.394 +0000 UTC"
        id                       = "1234567"
        jira_issue_transition_id = "31"
        merge_requests_events    = true
        password                 = (sensitive value)
        project                  = "1234567"
        project_key              = "PROJ"
        title                    = "Jira"
        updated_at               = "2020-04-02 10:41:05.69 +0000 UTC"
        url                      = "https://jira.url"
        username                 = "test@test.com"
    }
```

**Comment by wernerb** [comment]
@oboukili I will look at it, then its not picking up the state correctly remotely

**Comment by wernerb** [comment]
I made a pull request here to fix this: https://github.com/xanzy/go-gitlab/pull/829/files
Until accepted and a release is cut i will leave this PR open. I updated the code for the bugfix and will write some tests as well

**Comment by wernerb** [comment]
@ringods the pull request with the upstream fix got accepted. code fixed, tested all permutations also myself and modified existing test to include these three boolean flags. I had to bump go-gitlab to v0.30 to include the fix. The test for jira passes.

File: go.sum [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< Updated upstream
```
Comment (ringods) [inline go.sum:1]: @wernerb is this a valid line in a Go modules checksum file?

**Comment by ringods** [comment]
@wernerb the `master` branch is stable now. Can you rebase your branch? Please take the Go modules setup from `master`.

**Comment by wernerb** [comment]
> @wernerb I posted this update earlier this week:
> 
> [#133 (comment)](https://github.com/terraform-providers/terraform-provider-gitlab/issues/133#issuecomment-607454848)
> 
> I first want to have testsuites stable again. For this to happen, I also required go-gitlab 0.30.0 because it still contained a fix for the soft-delete. If my work in #291 makes the test suites stable, I will have to ask you to rebase once more. I will inform you here. Please have some patience.

Ofcourse!! 


> @wernerb the `master` branch is stable now. Can you rebase your branch? Please take the Go modules setup from `master`.

This is done, i dropped the commit updating to 0.30 go-gitlab!

**Comment by ringods** [comment]
Tnx!

**Pull request merged**